### PR TITLE
[site] Grammar fixes

### PR DESCRIPTION
**Pull request by JesterOrNot**
A few fixes.

### CODE REVIEW

File: src/pages/self-hosted.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const SelfHostedPage: React.SFC<{}> = () => (
                 title="Need Help Or Have Questions?"
                 text={
                     <React.Fragment>
-                        <p>Please reach out. We’re happy to help getting <br />the perfect Gitpod setup for your team.</p>
+                        <p>Please reach out. We’re happy to help get <br />your team the perfect Gitpod setup.</p>
```
Comment (nisarhassan12) [inline src/pages/self-hosted.tsx:72]: :+1:

File: src/utils/features.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ export const featuresCardsData = [
         icon: Terminal,
         iconText: <span>Linux<br />Terminals</span>,
         paragraphs: <React.Fragment>
-            <p>Gitpod instantly starts a machine in the cloud based on a Docker image. Tools that are required for your project or popular are readily installed and configured.</p>
+            <p>Gitpod instantly starts a machine in the cloud-based on a Docker image. Tools that are required for your project or popular are readily installed and configured.</p>
```
Comment (nisarhassan12) [inline src/utils/features.tsx:67]: Not sure whether this makes sense I think this was fine.
```suggestion
            <p>Gitpod instantly starts a machine in the cloud based on a Docker image. Tools that are required for your project or popular are readily installed and configured.</p>
```

File: src/utils/vendor.tsx [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
     },
     {
         title: "Show Your Products to Everyone",
-        paragraphs: ['A GitHub account is not required as Gitpod can be integrated with your own authentication system.', 'All workspaces are also sandboxed and hardened against abuses like spammers, miners, etc.']
+        paragraphs: ['A GitHub account is not required as Gitpod can be integrated with your authentication system.', 'All workspaces are also sandboxed and hardened against abuses like spammers, miners, etc.']
     },
     {
         title: "Guide Your Clients",
-        paragraphs: ['Let Gitpod show clients what to try next with an integrated helper. You can also integrate your own help system.', 'Gitpod guarantees an excellent user experience by eliminating the risk of a broken setup.']
+        paragraphs: ['Let Gitpod show clients what to try next with an integrated helper. You can also integrate your help system.', 'Gitpod guarantees an excellent user experience by eliminating the risk of a broken setup.']
```
Comment (nisarhassan12) [inline src/utils/vendor.tsx:16]: Was the use of `own` grammatically wrong? ^-^

Comment (ret394) [reply src/utils/vendor.tsx:16]: >risk of a broken setup

https://github.com/gitpod-io/website/pull/640 for feedback about gitpod usage.

thanks. have a nice day. keep up the great work. 👏

[ret394](https://github.com/ret394)

**Comment by jankeromnes** [comment]
Rebased, squashed, fixed conflict.

**Pull request merged**